datarepo-clienttests/src/main/java/runner/TestRunner.java

```diff
@@ -173,15 +173,20 @@ private void executeTestConfigurationNoGuaranteedCleanup() throws Exception {
     // Disruptive Thread: fetch script specification if config is defined
     if (config.disruptiveScript != null) {
       logger.debug("Creating thread pool for disruptive script.");
-      DisruptiveScript disruptiveScript = config.disruptiveScript.disruptiveScriptClassInstance();
-      disruptiveScript.setParameters(config.disruptiveScript.parameters);
+      DisruptiveScript disruptiveScriptInstance =
+          config.disruptiveScript.disruptiveScriptClassInstance();
+      disruptiveScriptInstance.setBillingAccount(config.billingAccount);
+      disruptiveScriptInstance.setServer(config.server);
+      disruptiveScriptInstance.setParameters(config.disruptiveScript.parameters);
 
       // create a thread pool for running its disrupt method
       disruptionThreadPool = (ThreadPoolExecutor) Executors.newFixedThreadPool(1);
 
-      DisruptiveThread disruptiveThread = new DisruptiveThread(disruptiveScript, config.testUsers);
+      DisruptiveThread disruptiveThread =
+          new DisruptiveThread(disruptiveScriptInstance, config.testUsers);
       disruptionThreadPool.execute(disruptiveThread);
       logger.debug("Successfully submitted disruptive thread.");
+      TimeUnit.SECONDS.sleep(1);
```

Comment:
Can you add a comment on why this sleep is necessary?

Reply:
Yes, it's just to give the disruptive script thread some time to start before kicking off the user journeys.